    bool find_obs_flag
    float64 dis
    int32 pos_left
    int32 pos_right
    int32 x_pos
    int32 y_pos